mltsp/run_script_in_container.py

```diff
@@ -10,26 +11,16 @@
     import argparse
     parser = argparse.ArgumentParser(description='MLTSP Docker scripts')
     parser.add_argument('--extract_custom_feats', action='store_true')
-    parser.add_argument('--featurize', action='store_true')
-    parser.add_argument('--build_model', action='store_true')
-    parser.add_argument('--predict', action='store_true')
     parser.add_argument('--tmp_dir', dest='tmp_dir', action='store', type=str)
     args = parser.parse_args()
 
     import sys
+    import os
     sys.path.append(args.tmp_dir)
+    sys.path.append(os.path.dirname(os.path.dirname(__file__)))
```

Comment:
Instead of calling dirname twice, a slightly cleaner syntax is ``os.path.join(os.path.dirname(__file__), '..')``